index.html

```diff
@@ -12,7 +12,16 @@
 <main>
   <header>
     <nav>
-      <div class="logo"><p>John Doe</p></div>
+      <div class="logo"><p>Tessy</p></div>
+      <ul class="menu-bar">
+        <li><a href="#frame" class="head-link">Portfolio</a></li>
+        <div class="indicator"></div>
+        <li><a href="#about" class="head-link">About</a></li>
+        <div class="indicator"></div>
+        <li><a href="#form-cont" class="head-link">Contact</a></li>
+        <div class="indicator"></div>
+      <li><a href="#" class="head-link icon-mess"><img src="./images/icon-mess.png" alt="main-image"></a></li>
+      </ul>
       <div class="hamburger">
         <span class="bar"></span>
```

Comment:
- [ ] please like the previous reviewer noted, kindly align your desktop header links to the right like in the figma version
your design
![Screenshot (659)](https://user-images.githubusercontent.com/65298048/206656092-5267086b-d947-4bb1-8bb3-72705ef61925.png)
figma
![Screenshot (660)](https://user-images.githubusercontent.com/65298048/206656122-da8ea239-4af4-4ff6-9642-5a4309e1c8aa.png)